tanner/utils/patterns.py

```diff
@@ -12,6 +12,7 @@
 PHP_CODE_INJECTION = re.compile(r'.*(;)*(echo|system|print|phpinfo)(\(.*\)).*')
 PHP_OBJECT_INJECTION = re.compile(r'(^|;|{|})O:[0-9]+:')
 CRLF_ATTACK = re.compile(r'.*(\r\n).*')
+XXE_INJECTION = re.compile(r'.*<(\?xml|(\!DOCTYPE.*)).*')
```

Comment:
I think `>` can be added in the end as well.

Reply:
Other than this it looks fine but I'll need 1-2 days to do some testing. So you can work on tests for the emulator.

Reply:
Awesome :)

Reply:
Should I add tests in new a PR?